bin/build-docs-site.sh

```diff
@@ -21,16 +21,16 @@ DIR=`dirname $0`
 REPO_ROOT=$DIR/..
 pushd $REPO_ROOT >/dev/null
 
-# Update apidocs
-lerna bootstrap
-lerna run --scope @loopback/docs prepack
-
 # Clean up sandbox/loopback.io directory
 rm -rf sandbox/loopback.io/
 
 # Shadow clone the `strongloop/loopback.io` github repo
 git clone --depth 1 https://github.com/strongloop/loopback.io.git sandbox/loopback.io
 
+# Update apidocs
+lerna bootstrap --scope @loopback/tsdocs
+lerna run --scope @loopback/docs prepack
```

Comment:
I don't see what difference it makes by moving the tsdocs generation to here.

Reply:
Biniam witnessed what is happening on my machine.
I work on an MD document.
Then I do `npm run build:site`
then I `cd sandbox/loopback.io`
then `npm start`

and when the server starts and I go to the locally hosted documentation page, and I go
to my document, the HTML does not contain my recent changes.

Reply:
@b-admike , please comment on what you saw happening on my machine and my generated HTML docs.

Reply:
Is the markdown file in `docs/site`?

Reply:
Pls try https://github.com/strongloop/loopback-next/pull/3120 if you still have issues.

Reply:
@raymondfeng the reason for moving the prepack script on docs here (not so much the tsdocs generation) is so that it runs after we git clone the `loopback.io` into sandbox so that we have it correctly symlinked IIUC.

Reply:
The `prepack` script on `@loopback/docs` generates apidocs and it's completely independent of `loopback.io`. The only prerequisite is to make sure all packages are built with `npm run build`.

Reply:
The `prepack` script also runs the `copy-readmes` script which is needed for the `loopback.io` clone. Btw https://github.com/strongloop/loopback-next/pull/3120 LGTM and we can try that out.

Reply:
Yes, but `copy-readmes` is also independent of the presence of `loopback.io`.

Reply:
> Biniam witnessed what is happening on my machine.
I work on an MD document.
Then I do npm run build:site
then I cd sandbox/loopback.io
then npm start

Can we reproduce the issue as follows?
1. Update `Context.md` in `docs/site`
2. `npm run build:site`
3. `cd sandbox/loopback.io`
4. `npm start`

Reply:
> Yes, but `copy-readmes` is also independent of the presence of `loopback.io`.

Good point. I thought there was a hard dependence of `loopback.io` on the `copy-readmes` to render changes in the readme files from local packages. 

> Can we reproduce the issue as follows?
> 
> Update Context.md in docs/site
> npm run build:site
> cd sandbox/loopback.io
> npm start
👍